lib/gallerist/models/album.rb

```diff
@@ -8,7 +8,7 @@ class Gallerist::Album < Gallerist::BaseModel
   self.table_name = 'RKAlbum'
 
   has_many :album_photos, primary_key: 'modelId', foreign_key: 'albumId'
-  has_one :key_photo, class: Gallerist::Photo, primary_key: 'posterVersionUuid', foreign_key: 'uuid'
+  has_one :key_photo, class_name: Gallerist::Photo, primary_key: 'posterVersionUuid', foreign_key: 'uuid'
```

Comment:
This is a change introduced in ActiveRecord 4.2.3 which wasn’t released at that time. It should have been `class_name` all the time, though. Thanks.